### DISCUSSION TITLE
More jinja?

### DISCUSSION THREAD

**Discussion by Friedrieck** [Mariusthvdb/custom-ui#106] [Ideas]
First, thank you so much, I eventually managed to have what I wanted (two attributes added to an entity, based on other entities' states).
It would have been so great to be able to write the templates in Jinja templates i.o. js, which I don't know very much...
Any chance to make it working with simple Jinja template instead of Javascript?

**Comment by Mariusthvdb** [comment]
No that won't be supported .
However there is an option to use it in template: 

Check the docs on the experimental just discovered ...
